src/styles/Display.module.css

```diff
@@ -0,0 +1,16 @@
+.list {
+  display: flex;
+  flex-direction: column;
+  width: 75rem;
+  height: 10.625rem;
+  margin: 2.313rem 2.25rem 0.938rem;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #fff;
+  background-color: #fff;
+  list-style: none;
+}
+
+.list .title {
+  background-color: aqua;
```

Comment:
- [ ] **These style changes Applies to your form to add a new book** Please correct the alignment of your form with the other book components above in order to match the design given 😃. Also, Please update the styles of your `submit` button too! Please look at the  comparison below 👇
Your design
![image](https://user-images.githubusercontent.com/90222110/226119260-467b799a-0186-4e42-8f25-42d545fe2715.png)
Given design
![image](https://user-images.githubusercontent.com/90222110/226119299-539798b4-c48a-4fa6-89f2-5b6772cc4217.png)